src/Tooltip/Tooltip.js

```diff
@@ -94,6 +94,9 @@ class Tooltip extends Util.mixin(BindMixin) {
     // the props.
     let anchor = state.anchor || props.anchor;
     let position = state.position || props.position;
+    let DOMNode = props.nodeType;
```

Comment:
hmm in our `Form` we don't have it as a capital. 

https://github.com/mesosphere/reactjs-components/blob/master/src/Form/Form.js#L325

what if you don't create a variable and just do `props.elementTag`